jenkins/ci.suse.de/cloud-mkcloud6.yaml

```diff
@@ -29,6 +29,8 @@
     version: 6
     previous_version: 5
     arch: x86_64
+    nodenumber_compute_ha: 4
+    controller: 2
```

Comment:
Can we also rename this to `nodenumber_controller`?

Reply:
Fixed